addons/l10n_id_efaktur/tests/test_l10n_id_efaktur.py

```diff
@@ -277,3 +279,304 @@ def test_efaktur_consume_code(self):
         self.assertEqual(self.efaktur.available, available_code - 1)
         # No error is raised when downloading.
         out_invoice_no_taxes.download_efaktur()
+
+    # =======================================
+    # E-faktur range management related tests
+    # =======================================
+
+    def test_change_min_max(self):
+        """ Test vlaue of available when changing min and max of range"""
+        self.assertEqual(self.efaktur.available, 10)
+
+        self.efaktur.min = "0000000000003"
+        self.assertEqual(self.efaktur.available, 8)
+
+        self.efaktur.max = "0000000000008"
+        self.assertEqual(self.efaktur.available, 6)
+
+        # update min and max together
+        self.efaktur.write({'min': "0000000000005", "max": "0000000000005"})
+        self.assertEqual(self.efaktur.available, 1)
+
+        self.efaktur.write({'min': "0000000000008", "max": "0000000000010"})
+        self.assertEqual(self.efaktur.available, 3)
+
+    def test_next_num(self):
+        """ Test if next_num computation is correct after altering availability """
```

Comment:
For tests like this one, please add some more context.
It's hard to ensure the test makes sense unless we go back to check what the range in self is.
It never hurts to do so to help future bugfixer understand faster what the test is doing, and what to expect

```suggestion
        """
        Test if next_num computation is correct after altering availability.
        
        Range: 0000000000001 - 0000000000010
        Base availability: 10
        """
```